src/main/java/com/spotify/github/v3/clients/GithubAppClient.java

```diff
@@ -34,6 +34,7 @@
 /** Apps API client */
 public class GithubAppClient {
 
+  private static final String GET_INSTALLATION_URL = "/app/installations/%s";
```

Comment:
Teeny tiny nitpick. Let's change the string name to clearly differentiate it from the other strings and explain its usage.

```suggestion
  private static final String GET_INSTALLATION_BY_ID_URL = "/app/installations/%s";
```